src/effects/VST3/VST3Effect.h

```diff
@@ -47,19 +51,12 @@ class VST3Effect final : public StatefulPerTrackEffect
    //Used if graphical plugin interface is disabled in the settings, or not provided by the plugin
    VST3ParametersWindow* mPlainUI { nullptr };
 
-   std::vector<std::unique_ptr<VST3Wrapper>> mRealtimeGroupProcessors;
```

Comment:
This commit loses the vector of processors in the effect, but does not restore that array elsewhere in the instance.